live-examples/html-examples/forms/css/button.css

```diff
@@ -0,0 +1,26 @@
+.styled {
+    border: 0;
+    line-height: 2.5;
+    padding: 0 20px;
+    font-size: 1rem;
+    text-align: center;
+    color: #ff0000;
+    text-shadow: 1px 1px 1px #000000;
```

Comment:
I am gonna by a nit picker and ask that you use the short form of the hex codes here, and the line above. ;)

Reply:
No probs — I also changed the text color to white, as the color contrast wasn't very good.

It's funny, I've always written hex codes out in full regardless; I think it harks back to a time when there were some mobile browsers than didn't support 3 character hexes.